extension/js/common/inject.ts

```diff
@@ -129,6 +129,7 @@ export class Injector {
           el.remove();
         })
       )
+      // eslint-disable-next-line @typescript-eslint/no-deprecated
       .hover(
```

Comment:
can we update `.hover` to `.on('hover', ..)` here?

Reply:
Yep!